process-document/src/main/java/com/ritense/processdocument/domain/impl/CamundaProcessInstanceId.java

```diff
@@ -19,14 +19,15 @@
 import com.ritense.processdocument.domain.ProcessInstanceId;
 import javax.persistence.Column;
 import javax.persistence.Embeddable;
+import java.io.Serializable;
 import java.util.Objects;
 import java.util.UUID;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentLength;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentNotNull;
 import static com.ritense.valtimo.contract.utils.AssertionConcern.assertArgumentTrue;
 
 @Embeddable
-public class CamundaProcessInstanceId implements ProcessInstanceId {
+public class CamundaProcessInstanceId implements ProcessInstanceId, Serializable {
```

Comment:
Is `Serializable` needed here?

Reply:
Yes, the JPA spec requires Id's to be serializable.